XbSymbolDatabase.h

```diff
@@ -447,6 +834,33 @@ static const char* xbe_type_str[XB_XBE_TYPE_MAX] = {
 typedef uint32_t xbaddr;
 #endif
 
+typedef struct _Library {
+    uint16_t    build_version;
+    uint16_t    qfe_version;
+    uint32_t    flag;
+    char        name[8];
+} Library;
+
+typedef struct _LibraryHeader {
+    unsigned int    count;
+    Library*        filters;
+} LibraryHeader;
+
+typedef struct _Section {
+    // These three variables must be fixed info.
+    char            name[8];
+    uint32_t        xb_virt_addr;
+    uint32_t        buffer_size;
+    void*           buffer_lower;
+} Section;
```

Comment:
These very generic names should get prefixes to avoid conflicts with other APIs / user-code.

Ideally this was true (and consistent) for *all* types / variables / functions in the public headers.